Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -1238,7 +1239,7 @@ end LocalHomeomorph
 
 open LocalHomeomorph
 
-variable [ChartedSpace H M] [ChartedSpace H' M']
+variable [CS : ChartedSpace H M] [CS' : ChartedSpace H' M']
```

Comment:
We don't need this either, right?

Reply:
Needed in `OpenEmbedding.singleton_smoothManifoldWithCorners`